scripts/upgrade

```diff
@@ -119,12 +127,26 @@ chown -R $app: $final_path
 chmod -R 755 $final_path
 
 #=================================================
-# PHP-FPM CONFIGURATION
+# SETUP APPLICATION WITH CURL
 #=================================================
-ynh_script_progression --message="Upgrading PHP-FPM configuration..." --weight=1
 
-# Create a dedicated PHP-FPM config
-ynh_add_fpm_config --package="$extra_php_dependencies"
+# Set the app as temporarily public for cURL call
+ynh_permission_update --permission "main" --add "visitors"
```

Comment:
You probably want to check if visitors is not in the permission, and if not, you probably want to remove it after that